geo/src/algorithm/simplifyvw.rs

```diff
@@ -207,7 +207,7 @@ fn vwp_wrapper<T>(
     epsilon: &T,
 ) -> Vec<Vec<Coordinate<T>>>
 where
-    T: CoordFloat + RTreeNum,
+    T: CoordFloat + RTreeNum + HasKernel,
```

Comment:
I guess this is technically a breaking change for those mythical users hypothetically using non-standard numeric types. I really expect these people don't currently exist, but I'd love to be proven wrong!

FWIW I think it's worth "breaking" to get the precision/robustness of our kernel implementations.

Reply:
Yeah I did wonder about that (show yourselves!), but as you say, I think it's worth it.